MyBooking.Repositories/RoleRepository.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MyBooking.Domain;
+using MyBooking.Domain.Entities;
+using MyBooking.Domain.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MyBooking.Repositories
+{
+    public class RoleRepository : IRoleRepository
+    {
+        protected readonly MyDbContext Context;
+        protected DbSet<Role> Entities;
+
+        public RoleRepository(MyDbContext context)
+        {
+            Context = context;
+            Entities = context.Set<Role>();
+        }
+
+        public List<Role> GetAll()
+        {
+            return Entities
+                .ToList();
+        }
+
+        public Role GetById(int roleId)
+        {
+            return Entities
+                .FirstOrDefault(f => f.Id == roleId);
+        }
+
+        public Role Insert(Role newRole)
```

Comment:
Do we need the option od adding new roles?

Reply:
Yes it is necessary at this level

Reply:
Okay, let it be